include/cvc5/cvc5_skolem_id.h

```diff
@@ -654,6 +654,33 @@ enum ENUM(SkolemId) : uint32_t
    * - Type: ``E``
    */
   EVALUE(SETS_MAP_DOWN_ELEMENT),
+  /**
+   * A skolem function that is unique per floating-point sort, introduced for
```

Comment:
Can you document the skolem indices and type of each of these (see the style of the other enum values).